examples/apps/collaborative-textarea/webpack.config.cjs

```diff
@@ -41,7 +41,7 @@ module.exports = (env) => {
 			},
 			plugins: [
 				new webpack.ProvidePlugin({
-					process: "process/browser",
+					process: "process/browser.js",
```

Comment:
Applied this change throughout the repo because the build was failing otherwise ([example](https://dev.azure.com/fluidframework/public/_build/results?buildId=290752&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=e51c1947-c90a-5cb4-a0bf-0143e1a3235f&l=1553)).

> The request 'process/browser' failed to resolve only because it was resolved as fully specified (probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"'). The extension in the request is mandatory for it to be fully specified. Add the extension to the request.

Which sounds like wherever we are currently adding that plugin in webpack config, either a) nothing "exercises it" (i.e. actually references process/browser) or b) anything that does, is not doing it from an ESM build.

It seems fine to always use the extension, I think.